README.md

```diff
@@ -18,6 +18,20 @@ Picture of Eleven
 npm i undici
 ```
 
+## Benchmarks
+
+Machine: Intel i7-7700k, 4.5GHz, 4C/8T, 32GB RAM, NVMe SSD
+Server: [H2O](https://github.com/h2o/h2o) 2.2.5
+Configuration: HTTP/1.1 without TLS
+
+```
+undici - request - pipe x 23,060 ops/sec ±1.11% (88 runs sampled)
+undici - stream  - pipe x 24,409 ops/sec ±1.70% (85 runs sampled)
```

Comment:
Put the fastest on top and then we are good! Thanks for the help!

Reply:
Done! :D